torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -951,15 +1078,12 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
     }
   }
 
-  KernelArgumentHolder kernel_arguments(options_.index_mode);
-  {
-    FUSER_PERF_SCOPE("ExecutorRunFusion::FillKernelArgStructure");
```

Comment:
Nit: I like the cleanliness of pushing these args at the same time. It just makes it really clear about the order of what's going into the arguments holder.